packages/vanilla/README.md

```diff
@@ -1,26 +1,31 @@
-# JSONForms - More Forms. Less Code
+# JSON Forms - More Forms. Less Code
+
 ### Complex Forms in the blink of an eye
 
-JSONForms eliminates the tedious task of writing fully-featured forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript.
+JSON Forms eliminates the tedious task of writing fully-featured forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript.
+
```

Comment:
There should be a quick start on which packages to install and a minimal example to run the vanilla renderers, see vue vanilla readme as a guideline.